src/fedora/40/amd64/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN dnf --setopt=install_weak_deps=False install -y \
 
 # Add MS package repo.
 COPY  microsoft.asc /tmp
-RUN dnf config-manager --add-repo https://packages.microsoft.com/fedora/39/prod/ && \
+RUN dnf config-manager --add-repo https://packages.microsoft.com/fedora/40/prod/ && \
```

Comment:
@MichaelSimons I noticed the F40 dockerfile was referencing the 39 repo. My guess is that wasn't intentional, but if it was I can change it back.

Reply:
This change looks correct.